tests/test_browser.py

```diff
@@ -3266,24 +3266,33 @@ def test_dynamic_link(self):
       }
     ''')
     open('side.cpp', 'w').write(r'''
-      #include <stdlib.h>
-      #include <string.h>
       char *side(const char *data);
+      char *side2(const char *data);
       char *side(const char *data) {
+        return side2(data);
+      }
+    ''')
+    open('side2.cpp', 'w').write(r'''
+      #include <stdlib.h>
+      #include <string.h>
+      char *side2(const char *data);
+      char *side2(const char *data) {
```

Comment:
This doesn't need to be declared twice here.

Reply:
I did it this way because it was already the case in this tests:

https://github.com/kripken/emscripten/blob/1.38.21-1-g46afd2308/tests/test_browser.py#L3268-L3276

and I thought it is maybe due to compiler (I recall hearing something long ago from a collegue that a C++ compiler insists to have prototypes for every functions). Anyway - after checking it seems not to be the case, so extra prototype for side2 will be gone.